Gemfile

```diff
@@ -28,7 +28,7 @@ gem "stimulus-rails"
 gem "jbuilder"
 
 # Use Redis adapter to run Action Cable in production
-# gem "redis", "~> 4.0"
+gem "redis", "~> 4.1.3"
```

Comment:
Why we need this gem? Please explain what this does, thanks!

Reply:
This one was for action cable purpose(during production) , i will remove it.